.circleci/config.yml

```diff
@@ -0,0 +1,124 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@5.0.0
+  slack: circleci/slack@4.4.2
+  browser-tools: circleci/browser-tools@1.2.4
+
+parameters:
+  ubuntu_machine:
+    type: string
+    default: 'ubuntu-2004:202111-02'
+  ubuntu_docker:
+    type: string
+    default: 'ubuntu:focal'
+
+workflows:
+  circleci_build_and_test:
+    jobs:
+      - code_style
+      - prettier
+      - build_and_test:
+          name: build_and_test_<< matrix.browser >>
+          context: slack-secrets
+          matrix:
+            parameters:
+              browser: ['node', 'chrome', 'firefox']
+      - generate_docs:
+          requires:
+            - code_style
+            - prettier
+            - build_and_test
+          filters:
+            branches:
+              only:
+                - master
+
+jobs:
+  code_style:
+    docker:
+      - image: << pipeline.parameters.ubuntu_docker >>
+    steps:
+      - checkout
+      - install_dependencies
+      - run: npm run lint
+
+  prettier:
+    docker:
+      - image: << pipeline.parameters.ubuntu_docker >>
+    steps:
+      - checkout
+      - install_dependencies
+      - run: |
+          npm install prettier
```

Comment:
This should not be needed since prettier should already be installed during `install_dependencies` -- in fact this may install a different version of the package

Reply:
updated ty